README.md

```diff
@@ -36,7 +36,6 @@ Git user attention
 
 3. After running `download-deps.py`.
 
-         $ cd cocos2d-x
```

Comment:
@pandamicro @andyque Thank you for your quick feedback!

If you just copy-paste the instructions from the readme you will get an error:

    oscr@oscr-VirtualBox:~/temp/cocos2d-x$ python download-deps.py
    (...)
    oscr@oscr-VirtualBox:~/temp/cocos2d-x$ cd cocos2d-x
    bash: cd: cocos2d-x: No such file or directory

This actually caused me some confusion when I got started. If we want the user to know where (s)he should be when a command is executed then maybe we could hint this by letting the command prompt show the current catalog? For example:

    $ git clone https://github.com/cocos2d/cocos2d-x.git
    $ cd cocos2d-x
    cocos2d-x $ python download-deps.py
    cocos2d-x $ git submodule update --init